src/extension.ts

```diff
@@ -13,9 +13,15 @@ export function activate(context: ExtensionContext) {
   let skipCommand = commands.registerCommand('gmusic.skip', () => {
     gMusic.forward()
   })
+
```

Comment:
Let's keep the line breaks consistant. For commands, I'd like to keep them without.